arcgis-ios-sdk-samples/Route & Directions/Navigate route/NavigateRoute.storyboard

```diff
@@ -0,0 +1,80 @@
+                                <rect key="frame" x="0.0" y="0.0" width="375" height="20.5"/>
+                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="0.59999999999999998" colorSpace="custom" customColorSpace="sRGB"/>
+                                <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                <nil key="textColor"/>
+                                <nil key="highlightedColor"/>
+                            </label>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="iSf-vt-cp3">
+                                <rect key="frame" x="0.0" y="623" width="375" height="44"/>
+                                <items>
+                                    <barButtonItem enabled="NO" title="Navigate" id="Pj1-2W-66g">
+                                        <connections>
+                                            <action selector="startNavigation" destination="JxV-EA-CYk" id="tmV-vR-qVK"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="nyt-vH-6dO"/>
+                                    <barButtonItem enabled="NO" title="Recenter" id="yMb-Zf-aou">
+                                        <connections>
+                                            <action selector="recenter" destination="JxV-EA-CYk" id="klD-Z0-Uhp"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="1EW-wh-IEu"/>
+                                    <barButtonItem enabled="NO" title="     Reset" id="W0g-he-t8F">
+                                        <connections>
+                                            <action selector="reset" destination="JxV-EA-CYk" id="6P0-Qg-upN"/>
+                                        </connections>
+                                    </barButtonItem>
+                                </items>
+                            </toolbar>
+                        </subviews>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <constraints>
+                            <constraint firstItem="EC7-F9-vN9" firstAttribute="top" secondItem="YVM-3B-TME" secondAttribute="top" id="2Vh-u4-Cba"/>
+                            <constraint firstItem="IhO-bQ-arF" firstAttribute="leading" secondItem="YVM-3B-TME" secondAttribute="leading" id="Kxc-Jh-jye"/>
```

Comment:
The background now extends into the safe area, but so does the text. The text should be within the safe area, just like the attribution bar at the bottom of the map view:

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/2257493/78369223-b405e800-7579-11ea-91ff-6e52f7360b90.png">

That is why I suggested having two views: one for the background that is edge-to-edge and one for the text.

Reply:
Hmm, that is really a great point. I'll do a bit research on that. Also later I'll open another PR to fix the other examples with the same problem.

Reply:
For the Connected Trace example, it embeds the UILabel in a stack view (seems that you implemented that one). I think that would be a good practice to follow.

Reply:
To unify the styles in current examples, proposed #845 . That PR will make the `UILabel` extend into safe area. The text in the labels are not explicitly aligned to safe area, because their widths are short and are all center aligned.